sorcery.gemspec

```diff
@@ -1,37 +1,63 @@
-lib = File.expand_path('../lib', __FILE__)
```

Comment:
@youzik please revert all the changes in this file, they're not needed and some of them break the current way the gem is configured (e.g. it hardcodes the version of the gem instead of loading it from `Sorcery::VERSION`)